app.js

```diff
@@ -54,16 +55,24 @@ function displayRandomProductImages() {
         randomProductImage3 = getRandomProduct(sliceOfProductData);     
     }
 
-    //render the random images in the browser as radio buttons
+    //render the random products in the browser as radio buttons
     radioButtonOne.value = randomProductImage1.id;
     radioButtonTwo.value = randomProductImage2.id;
     radioButtonThree.value = randomProductImage3.id;
 
-    //render the name of the image in the span of the radio button
+    //render the image of the product in the browser as the radio button
+    radioButtonOne.input = randomProductImage1.image.src;
+    radioButtonTwo.input = randomProductImage2.image.src;
+    radioButtonThree.input = randomProductImage3.image.src;
+
+
+    //render the name of the product in the span of the radio button
     productOneSpan.textContent = randomProductImage1.name;
     productTwoSpan.textContent = randomProductImage2.name;
     productThreeSpan.textContent = randomProductImage3.name;
 
```

Comment:
all of this looks great, this is also where you're going to want to change the src of your images. 
something like` image1.src = randomProductImage1.image;`